packages/bff/src/model/api/apiConverter.ts

```diff
@@ -167,3 +167,19 @@ export function toTenantWithOnlyAttributes(
     attributes: tenant.attributes.map(toTenantAttribute).flat(),
   };
 }
+
+export function toCatalogEServiceSeed(
```

Comment:
```suggestion
export function toCatalogCreateEServiceSeed(
```
I think specifying what kind of seed it is would help us in the future